src/components/Integrations/IntegrationsDrawer.tsx

```diff
@@ -0,0 +1,171 @@
+import React, { useEffect } from 'react';
+import {
+  DrawerActions,
+  DrawerCloseButton,
+  DrawerHead,
+  DrawerPanelContent,
+  Tab,
+  TabTitleText,
+  Tabs,
+  Title,
+} from '@patternfly/react-core';
+import {
+  EventTypes,
+  useDataViewEventsContext,
+} from '@patternfly/react-data-view';
+import {
+  Dropdown,
+  DropdownItem,
+  DropdownList,
+  MenuToggle,
+  MenuToggleElement,
+} from '@patternfly/react-core';
+import EllipsisVIcon from '@patternfly/react-icons/dist/dynamic/icons/ellipsis-v-icon';
+import IntegrationDetails from './IntegrationDetails';
+import { IntegrationRow } from './Table';
+import messages from '../../properties/DefinedMessages';
+import { useIntl } from 'react-intl';
+
+const ActionDropdown: React.FunctionComponent = () => {
+  const [isOpen, setIsOpen] = React.useState(false);
+
+  const intl = useIntl();
+
+  const onToggleClick = () => {
+    setIsOpen(!isOpen);
+  };
+
+  const onSelect = () => {
+    setIsOpen(false);
+  };
+
+  return (
+    <Dropdown
+      isOpen={isOpen}
+      onSelect={onSelect}
+      popperProps={{ position: 'right', enableFlip: true }}
+      onOpenChange={(isOpen: boolean) => setIsOpen(isOpen)}
+      toggle={(toggleRef: React.Ref<MenuToggleElement>) => (
+        <MenuToggle
+          ref={toggleRef}
+          aria-label="kebab dropdown toggle"
+          variant="plain"
+          onClick={onToggleClick}
+          isExpanded={isOpen}
+        >
+          <EllipsisVIcon />
+        </MenuToggle>
+      )}
+      shouldFocusToggleOnSelect
+    >
+      <DropdownList>
+        <DropdownItem
+          value={0}
+          key="action"
+          description={intl.formatMessage(messages.pauseDescription)}
+        >
+          {intl.formatMessage(messages.integrationdropdownPause)}
```

Comment:
We should toggle between pause and `enabled`. If the integration is paused we should show to cusomers option to enable this integration.